packages/swarm-components/src/TogglePill.test.jsx

```diff
@@ -29,6 +29,12 @@ describe('TogglePill', () => {
 				Full width (grow)
 			</TogglePill>,
 		],
+		[
```

Comment:
We should also add an example of the reset state to the docs. See `packages/swarm-docs/src/components/toggleExample.js`. To view the docs, run `yarn develop` in `packages/swarm-docs/` and follow the link.

Reply:
Ah thanks for pointing this out, have added to `togglePillExample`